packages/plugin-ext/src/plugin/quick-open.ts

```diff
@@ -576,6 +577,15 @@ export class InputBoxExt extends QuickInputExt implements theia.InputBox {
         this.update({ prompt });
     }
 
+    get valueSelection(): readonly [number, number] | undefined {
+        return this._valueSelection;
+    }
+
+    set valueSelection(valueSelection: readonly [number, number] | undefined) {
+        this._valueSelection = valueSelection ?? [0, this.value.length];
```

Comment:
I don't believe we need to set a default value for this anymore, right?

Reply:
this is specified in the vs code api docs that setting undefined is supposed to do this

Reply:
_When undefined the whole pre-filled value will be selected_

Reply:
I believe the underlying `IQuickInput` element will take care of this by itself. We don't need to explicitly set the value, especially since the following will yield unexpected results:

```
quickOpen.value = '';
quickOpen.valueSelection = undefined;
quickOpen.value= 'x';
```

Reply:
ok I'll change and test it again